Common++/header/IpAddress.h

```diff
@@ -65,7 +72,12 @@ namespace pcpp
 		/**
 		 * Returns a pointer to 4-byte array representing the IPv4 address
 		 */
-		const uint8_t* toBytes() const { return m_Bytes; }
+		const uint8_t* toBytes() const { return m_Bytes.data(); }
+
+		/**
+		 * Returns a reference to a 4-byte standard array representing the IPv4 address
+		 */
```

Comment:
Maybe we can do:
```cpp
/**
 * @return A reference to a 4-byte standard array representing the IPv4 address
 */
```

We can also update the documentation of `toBytes()`

Reply:
Updated the documentation b3228c1, please check it and see if its good.

Reply:
In getters we usually tend to avoid the description and only have a `@return` clause. For example;
```cpp
/**
 * @return A reference to a 4-byte standard array representing the IPv4 address
 */
```

Instead of:
```cpp
/**
 * Returns a reference to a 4-byte standard array representing the IPv4 address
 * @return A reference to a 4-byte standard array representing the IPv4 address
 */
```

Reply:
I guess it makes sense for the getters, tho I was mostly following the same pattern as the `toString` method directly below the `toBytes`, as I consider the methods more similar to the `toString` converter method then to a plain getter method.

Reply:
Yes, we changed it at some point because it doesn't make a lot of sense to repeat the same sentence twice... feel free to change `toString()` as well if you want 🙂

Reply:
Fixed 1d145b7